mod/kalvidassign/db/upgrade.php

```diff
@@ -85,5 +85,18 @@ function xmldb_kalvidassign_upgrade($oldversion) {
         upgrade_mod_savepoint(true, 2014023000.01, 'kalvidassign');
     }
 
+    if ($oldversion < 2022041905) {
+        $table = new xmldb_table('kalvidassign');
+        $field = new xmldb_field('hidegrader', XMLDB_TYPE_INTEGER, 1, true, true, null, 0, 'timemodified');
+
+        // Conditionally launch add field metadata.
+        if (!$dbman->field_exists($table, $field)) {
+            $dbman->add_field($table, $field);
+        }
+    }
+
+    // Kalvidassign savepoint reached.
+    upgrade_mod_savepoint(true, 2014023000.01, 'kalvidassign');
```

Comment:
@gonenradai  do you know what the value for this savepoint should be?

Reply:
I think this is some indication of the "version" of the DB schema of the module.
That way, if a module needs to update the schema, it can compare the last known save point that is written in the DB and know which steps to perform (for example, if the upgrade is jumping few versions and needs multiple ALTER calls) to upgrade to the latest schema.

Reply:
I think the value you want to save is the new version that you're comparing against - 2022041905